python/ray/autoscaler/sdk.py

```diff
@@ -167,7 +168,7 @@ def get_worker_node_ips(cluster_config: Union[dict, str]) -> List[str]:
         return commands.get_worker_node_ips(config_file)
 
 
-def request_resources(num_cpus=None, bundles=None):
+def request_resources(num_cpus=None, bundles=None) -> None:
```

Comment:
Should be the same types as commands.py I think.

Reply:
Yes, forgot about it.